src/msg_data.c

```diff
@@ -310,15 +362,27 @@ MessageCharset* MsgCharsets[] = {
 };
 
 s32 MsgLetterRasterOffsets[] = {
+#if VERSION_IQUE
+    0x00056E90, 0x00058A90, 0x0005A690, 0x0005C290,
```

Comment:
These should probably just be specified as offsets to the images/palettes.

Reply:
done with it, except for the palette offsets which the linker doesn't acknowledge alignment for the offset address.
```
                0x000000000000eb72                charset_letter_content_1_pal_OFFSET = .
                0x000000000000eb72                charset_letter_content_1_pal = .
 ver/us/build/assets/us/charset/letter_content_1.pal.o(.data*)
 *fill*         0x000000000000eb72        0x6 00000000
 .data          0x000000000000eb78      0x200 ver/us/build/assets/us/charset/letter_content_1.pal.o
                0x000000000000eb78                _binary_ver_us_build_assets_us_charset_letter_content_1_pal_bin_start
                0x000000000000ed78                _binary_ver_us_build_assets_us_charset_letter_content_1_pal_bin_end
```

Reply:
can you put a // TODO palette offsets on it so we don't forget?